samples/dashboards/demos/axes-sync/demo.css

```diff
@@ -1,5 +1,5 @@
-@import "https://code.highcharts.com/css/highcharts.css";
-@import "https://code.highcharts.com/css/dashboards/gui.css";
+@import url("https://code.highcharts.com/css/highcharts.css");
+@import url("https://code.highcharts.com/css/dashboards/gui.css");
```

Comment:
It isn't fully solved.
![image](https://github.com/highcharts/highcharts/assets/57004120/f81dfffe-8b8e-4ed7-92e6-6fcdfa451d8f)